doc/flamegraphs.md

```diff
@@ -0,0 +1,108 @@
+## Flame Graphs
+
+Flame graphs are a way of visualizing software CPU and heap profiles. Any tool
+that can create stack data in the correct format can be used to generate a flame
+graph. You can learn more about flame graphs
+[here](http://www.brendangregg.com/flamegraphs.html).
+
+Example flame graph (click to zoom):
```

Comment:
When I preview this page on Github, clicking on the image takes me to the image page, rather than zoom. An animated gif might be a better way to illustrate the behavior.